doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,5 +1,7 @@
 # Upgrade Notes
 
+## 6.2.3
```

Comment:
next one will 6.3.0

Reply:
see https://github.com/pimcore/pimcore/milestones

Reply:
but its already present in 6.2.3

Reply:
@see https://github.com/pimcore/pimcore/issues/4740

Reply:
since 6.1.0 ?

Reply:
ohh yes sry missread that - moved it down to 6.1.0